### PR TITLE
Update `broccoli-babel-transpiler` to v8.0.0-beta.1

### PR DESCRIPTION
**Pull request by bertdeblock**

### CODE REVIEW

**Comment by kellyselden** [comment]
Is it expected that this releases in a beta version as well?

**Comment by bertdeblock** [comment]
Not 100% sure, but I would think so yes.

**Comment by kellyselden** [comment]
Sounds good to me.

**Review by kellyselden — APPROVED** [review]

File: index.js [hunk 1]
```diff
@@ -91,7 +92,7 @@ module.exports = {
       return options;
     } else {
       // legacy codepath
-      return Object.assign({}, this._buildBroccoliBabelTranspilerOptions(config), options);
+      return merge({}, this._buildBroccoliBabelTranspilerOptions(config), { babel: options });
```
Comment (rwjblue) [inline index.js:95]: Can we just do spread here (instead of pulling `lodash.merge`)

File: index.js [hunk 2]
```diff
@@ -148,7 +148,7 @@ module.exports = {
     let config = _config || this._getAddonOptions();
     let description = `000${++count}`.slice(-3);
     let postDebugTree = this._debugTree(inputTree, `${description}:input`);
-    let options = Object.assign({}, this._buildBroccoliBabelTranspilerOptions(config), this.buildBabelOptions('babel', config));
+    let options = merge({}, this._buildBroccoliBabelTranspilerOptions(config), { babel: this.buildBabelOptions('babel', config) });
```
Comment (rwjblue) [inline index.js:151]: Same here, I'd like to avoid `merge`

**Review by rwjblue — COMMENTED** [review]
Just a couple of minor points here, but overall this LGTM

**Review by rwjblue — APPROVED** [review]

**Pull request merged**